ui/debug/src/js/soundEffects.js

```diff
@@ -0,0 +1,84 @@
+const {isFunction} = require('lodash');
```

Comment:
If you want to import single function from Lo-Dash, you should do
```JavaScript
const isFunction = require('lodash/isFunction');
```
instead, because this is going to load the entire library.